src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+function toNumber(string) {
+  const searchNumber = string.match(/[0-9]/g);
+
+  if (searchNumber) {
+    return Number(searchNumber.join(''));
+  }
+
+  return string;
+}
+
+table.addEventListener('click', event => {
+  const target = event.target;
+
+  if ([...table.rows[0].cells].includes(target)) {
+    const findIndex = [...table.rows[0].cells].indexOf(target);
+
```

Comment:
table.rows[0] it's a table header, use table.tHead instead 
and create a separate var for it, you will need to use it several times

Reply:
read about .cellIndex (you can use this prop instead of .indexOf array method 
https://learn.javascript.ru/dom-navigation#dom-navigation-tables